controller/controller.go

```diff
@@ -28,6 +29,18 @@ var AllExtensionsMap = map[string]extension.Extension{}
 // ExtensionUpdaterTimeout is the amount of time to wait between getting new updates from DynamoDB for the list of extensions
 var ExtensionUpdaterTimeout = time.Minute * 10
 
+//IsJSONRequest is used to check if JSON parser should be used
+func IsJSONRequest(data []byte) bool {
+	var js map[string]interface{}
+	return json.Unmarshal(data, &js) == nil
+}
+
+//IsJSONQuery is used to check if JSON parser should be used
+func IsJSONQuery(version string) bool {
```

Comment:
For `WebStoreUpdateExtensions` we use the `prodversion` parameter in the query string to figure out if the JSON parser should be used.

Reply:
Is it not better to use the Content-Type header?